h/migrations/versions/534754b1bf54_add_nipsa_table.py

```diff
@@ -0,0 +1,25 @@
+"""Add the nipsa table.
+
+Revision ID: 534754b1bf54
+Revises: 49764e8a7e69
+Create Date: 2015-07-17 12:51:53.582010
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '534754b1bf54'
+down_revision = '49764e8a7e69'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.create_table(
+        'nipsa',
+        sa.Column('user_id', sa.UnicodeText, primary_key=True, index=True)
```

Comment:
We're consistent in the rest of the code base about this being called `userid`, without an underscore. It should be the same here (and in the corresponding model).